dist/immutable-nonambient.d.ts

```diff
@@ -127,18 +127,52 @@
 
     /**
      * True if the provided value is a List
+     *
+     * ```js
+     * List.isList([]); // false
+     * List.isList(List([])); // true
+     * ```
      */
     function isList(maybeList: any): boolean;
 
     /**
      * Creates a new List containing `values`.
+     *
+     * ```js
+     * List.of(1, 2, 3, 4).toJS();
+     * // [ 1, 2, 3, 4 ]
+     *
+     * List.of({x:1}, 2, [3], 4).toJS();
```

Comment:
seems like we could get by with just the first example - this one is a bit confusing because did `{x:1}` get immutablized?